test/asm_test/X86/raise-cvt.s

```diff
@@ -0,0 +1,117 @@
+// REQUIRES: x86_64-linux
+// RUN: clang -O0 -o %t %s
+// RUN: llvm-mctoll -d -I /usr/include/stdio.h %t
+// RUN: clang -o %t-dis %t-dis.ll
+// RUN: %t-dis 2>&1 | FileCheck %s
+// CHECK: 1.5
```

Comment:
Appears that the output check is incorrect. Instead, the following change is needed? Please confirm.

```
diff --git a/test/asm_test/X86/raise-cvt.s b/test/asm_test/X86/raise-cvt.s
index fd99789..b6bbb37 100644
--- a/test/asm_test/X86/raise-cvt.s
+++ b/test/asm_test/X86/raise-cvt.s
@@ -3,7 +3,8 @@
 // RUN: llvm-mctoll -d -I /usr/include/stdio.h %t
 // RUN: clang -o %t-dis %t-dis.ll
 // RUN: %t-dis 2>&1 | FileCheck %s
-// CHECK: 1.5
+// CHECK: 42.0 42.0 42.0 42.0 42 42 42 42
+// CHECK-NEXT: 42.0 42.0 42.0 42.0 42 42 42 42
 // CHECK-EMPTY
 
 .text
```

Reply:
Never mind. Applied the change and pushed.